library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -179,6 +179,11 @@
     private ScrollListener mScrollListener;
     private AddEventClickListener mAddEventClickListener;
 
+    //Scroll by amount of days desired functionnality
+    private boolean mIsCustomScrollableAmountsOfDaysEnabled = false;
+    private float startOriginForScroll = 0;
```

Comment:
What is the difference and relation with `mCurrentOrigin.x` ?

Reply:
The current origin.x is updated when I started to scroll. 
For exemple I start à 0 on x axis.
I start a scroll and now I'am at -200 (Scroll Direction Left)
The mCurrentOrigin.x value is -200.0, and startOriginForScroll will stay at 0. I've done that because I have added in my own repo the functionnality "safe scrolling" that let you go back to the latest valid position you had.

Reply:
ok, but wouldn't it be possible without this variable ? If it is possible, please remove this extra variable